legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -529,21 +539,30 @@ Payment Activities are created when a Payment Request has been **created**, **pa
     }
   ],
   "lineItems": [
-      {
-        "name": "Coffee Grounds",
-        "sku": "GH1234",
-        "qty": "1",
-        "price": "4195",
-        "tax": "15.00",
-      },
-      {
-        "name": "Centrapay Cafe Mug",
-        "sku": "SB456",
-        "qty": "25",
-        "price": "1995",
-        "tax": "15.00",
-        "discount": "199",
-      },
+    {
+      "name": "Coffee Grounds",
+      "sku": "GH1234",
+      "qty": "1",
+      "price": "4195",
+      "tax": "15.00",
+    },
+    {
+      "name": "Centrapay Cafe Mug",
+      "sku": "SB456",
+      "qty": "25",
+      "price": "1995",
+      "tax": "15.00",
+      "discount": "199",
+    },
+    {
+      "amount": "6190",
+      "assetType": "centrapay.token.test",
+      "collections": [
+        {
+          "id": "345224"
+        }
+      ]
```

Comment:
Is this in the wrong place? I would expect this to be in the `paymentOptions` array.

Reply:
Yup